Packs/PAN-OS/Integrations/Panorama/Panorama.yml

```diff
@@ -7466,6 +7466,195 @@ script:
+      type: String
+      description: Host ID
+    - contextPath: PANOS.SystemStatus.up
+      description: Whether the host device is up or still unavailable.
+      type: String
+  - arguments:
+    - default: false
+      description: ID of host (serial or hostname) to update the state.
+      isArray: false
+      name: hostid
+      required: true
+      secret: false
+    - default: false
+      description: New state.
+      isArray: false
+      name: state
+      required: true
+      secret: false
+    deprecated: false
+    description: Checks the status of the given device, checking whether it's up or
+      down and the operational mode normal
+    name: pan-os-platform-update-ha-state
+    outputs:
+    - contextPath: PANOS.HAStateUpdate.hostid
+      type: String
+      description: Host ID
+    - contextPath: PANOS.HAStateUpdate.state
+      description: New HA State
+      type: String
+  - arguments:
+    - default: false
+      defaultValue: null
+      description: String to filter to only check given device
+      isArray: false
+      name: device_filter_string
+      required: false
+      secret: false
+    deprecated: false
+    description: Checks at least one Log forwarding profile is configured according to best practices.
+    name: pan-os-hygiene-check-log-forwarding
+    outputs:
+    - contextPath: PANOS.ConfigurationHygiene.Summary.description
+      description: The description of the check
+      type: String
+    - contextPath: PANOS.ConfigurationHygiene.Summary.issue_code
+      description: The shorthand code for this hygiene check
+      type: String
+    - contextPath: PANOS.ConfigurationHygiene.Summary.result
+      description: Whether the check passed or failed
+      type: String
+    - contextPath: PANOS.ConfigurationHygiene.Summary.issue_count
+      description: Total number of matching issues
+      type: String
+    - contextPath: PANOS.ConfigurationHygiene.Result.hostid
+      type: String
+    - contextPath: PANOS.ConfigurationHygiene.Result.container_name
+      description: What parent container (DG, Template, VSYS) this object belongs
+        to.
+      type: String
+    - contextPath: PANOS.ConfigurationHygiene.Result.issue_code
+      description: The shorthand code for the issue
+      type: String
+    - contextPath: PANOS.ConfigurationHygiene.Result.description
+      description: Human readable description of issue
+      type: String
+    - contextPath: PANOS.ConfigurationHygiene.Result.name
+      description: The affected object name
+      type: String
+  - arguments:
+    - default: false
+      defaultValue: null
+      description: String to filter to only check given device
+      isArray: false
+      name: device_filter_string
+      required: false
+      secret: false
+    - default: false
+      defaultValue: critical,high
+      description: csv list of severities that must be in drop/reset/block-ip mode.
+      isArray: false
+      name: minimum_block_severities
```

Comment:
does it have specific values that it can have? if yes use a predefined section

Reply:
It does not, there's only a set number of alert levels but the command only takes them in csv format (because you need to be able to specify multiple at once). As far as I am aware you can't use a list as a predefined argument so this is the only way we can do it.